### PR TITLE
[montilang/en] Add language MontiLang

### PR DESCRIPTION
**Pull request by lduck11007**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
@vendethiel @adambard This looks good to me, but I'm not familiar with the language so I'd like to have your review before merging it in.

File: montilang.html.markdown [hunk 1]
```diff
@@ -0,0 +1,234 @@
+---
+
+language: "MontiLang"
```
Comment (divayprakash) [inline montilang.html.markdown]: Please remove the blank line preceding this and also change this to `language: montilang`

Comment (lduck11007) [reply montilang.html.markdown]: I've just done this.

**Comment by lduck11007** [comment]
Website landing page is at http://montilang.ml/

**Comment by divayprakash** [comment]
@lduck11007 Thanks for the contribution and welcome to the community!

**Comment by lduck11007** [comment]
Thanks

**Pull request merged**